.vscode/settings.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Can you remove `.vscode` directory and add it to `.gitgnore`?

Reply:
I would argue for keeping it in as it just enforces the settings needed for it to pass linting. Line endings, indentation, etc. Maybe the search excludes are less necessary, I'll remove those

Reply:
Right, but `.vscode` is just for vscode. If this was adding a `.editorconfig` file it'd be one thing but I'd rather not have anything that is not editor agnostic in the repo.